src/community/design-system-day-2022/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Watch videos from Design System Day 2022
```

Comment:
Will need to have a think about the 2 page titles, as we'll want to give 2023 a lot more prominence and avoid repetition. Perhaps reorder in sidebar menu?